spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+
+Shawn Bayern and Hans Bergsten deserve special
+credit for being actively involved in all design issues of this
+specification. Their vast expertise and commitment to excellence has had
+a profound impact in every single aspect of this specification. Mille
+mercis Shawn et Hans! Don't know how we would have done it without you
+two.
+
+Many thanks to Jan Luehe for taking ownership of
+the internationalization and formatting chapters of this specification
+on short notice, and doing an incredible job.
+
+Special mention to Nathan Abramson for being a
+driving force behind the expression language introduced in JSTL, to
+James Strachan for leading the group in our understanding of XML for
+page authors, and to Craig McClanahan for his help on servlet and J2EE
+platform related issues.
+
+This specification has drawn a lot of its design
+ideas from pioneers in the field of tag libraries. We are grateful to
+the Jakarta project at Apache, as well as other efforts in the industry,
+where projects led by Craig McClanahan (Struts), James Strachan (XTags),
+Morgan Delagrange (DBTags), Tim Dawson (I18N), Glenn Nielsen (many
+utility taglibs), Scott Hasse (JPath), Dmitri Plotnikov (JXPath), Pasi
+Salminen (O&D Struts), have greatly influenced the design of the JSTL
+libraries.
+
+The RI team composed of Shawn Bayern (lead),
+Nathan Abramson, Justyna Horwat, and Jan Luehe has done a wonderful job
+at turning code faster than the specification could be written.
+
+Quality has been in the capable hands of Ryan
+Lubke, lead of the TCK team that also includes Lance Andersen. David
+Geary’s help in doing thorough reviews of the specification was also
+greatly appreciated.
+
+We are also grateful to the product team at Sun
+Microsystems who helped us sail efficiently through this specification:
+Jim Driscoll, Karen Schaffer, George Grigoryev, Stephanie Bodoff,
+Vanitha Venkatraman, Prasad Subramanian, and Xiaotan He.
+
+Finally, we'd like to thank the community at
+large for their ever increasing interest in this technology. We sure
+hope you’ll enjoy the JSP Standard Tag Library.
+
+== Comments
+
+We are interested in improving this
+specification and welcome your comments and suggestions. You can email
+your comments to us at:
+
+ jstl-dev@eclipse.org
+
+:sectnums:
 == Introduction
 
-Jakarta Standard Tag Library provides a specification document, API and TCK that extends the 
-Jakarta Server Pages specification by adding a tag library of JSP tags for common tasks, 
-such as XML data processing, conditional execution, database access, loops and internationalization. 
+Jakarta Standard Tag Library provides a specification document, API and TCK that extends the
+Jakarta Server Pages specification by adding a tag library of JSP tags for common tasks,
+such as XML data processing, conditional execution, database access, loops and internationalization.
+
+
+=== Goals
+
+The ultimate goal of JSTL is to help simplify
+JavaServer™ Pages (JSP™) page authors’ lives.
```

Comment:
Replace JavaServer pages, remove JSP reference

Reply:
The ultimate goal of the Jakarta Standard Tag Library is to help simplify Jakarta Server Pages authors’ lives.